website/docs/r/vapp_network.html.markdown

```diff
@@ -3,13 +3,12 @@ layout: "vcd"
 page_title: "vCloudDirector: vcd_vapp_network"
 sidebar_current: "docs-vcd-resource-vapp-network"
 description: |-
-  Provides a vCloud Director vApp isolated Network. This can be used to create and delete internal networks for vApps to connect.
+ Allows to provision vApp network and optionally connect it to existing Org VDC network.
 ---
 
 # vcd\_vapp\_network
 
- Provides a vCloud Director vApp isolated Network. This can be used to create and delete internal networks for vApps to connect.
- This network is not attached to external networks or routers.
+ Allows to provision vApp network and optionally connect it to existing Org VDC network.
```

Comment:
We need an example below how to "connect it to existing Org VDC network". What about adding these lines to the example below:

```

  # Comment below line to create an isolated vApp network
  org_network_name = "my-org-network"

```

BTW, please polish from "#Optional" to "# Optional" in the example too.

Reply:
updated